src/tests/ftest/nvme/nvme_health.py

```diff
@@ -34,21 +34,21 @@ def test_monitor_for_large_pools(self):
         # pylint: disable=attribute-defined-outside-init
         # pylint: disable=too-many-branches
         no_of_pools = self.params.get("number_of_pools", '/run/pool/*')
-        # Stop the servers to run SPDK too to get the server capacity
-        self.stop_servers()
-        storage = self.get_nvme_max_capacity()
-        self.start_servers()
+        pool_capacity = self.params.get("pool_used_percentage", '/run/pool/*')
+        pool_capacity = pool_capacity / 100
+        storage = self.get_max_storage_sizes()
 
-        # Create the pool from 80% of available of storage space
-        single_pool_nvme_size = int((storage * 0.80)/no_of_pools)
+        #Create the pool from available of storage space
+        single_pool_nvme_size = int((storage[1] * pool_capacity)/no_of_pools)
+        single_pool_scm_size = int((storage[0] * pool_capacity)/no_of_pools)
 
         self.pool = []
         # Create the Large number of pools
         for _pool in range(no_of_pools):
+            self.log.info("-- Creating pool number = {}".format(_pool))
```

Comment:
(pylint-logging-format-interpolation) Use lazy % formatting in logging functions